PACK_PCB.md

```diff
@@ -37,8 +37,8 @@ Connections for the pack should be made according to the table below.
 
 For connecting the UART pins, use a suitable FTDI chip such as the same **FTDI Basic 5V** chip used for programming the WavTrigger. Connect each pin to the respective label to the FTDI chip.
 
-- If using an FTDI to USB cable, the RTS pin will connect to the DTR board pin.
-- The CTS pin on any FTDI device will not be used (connects to GND on the board).
+- If using an FTDI to USB cable, the RTS pin will connect to the DTR pin on the PCB.
+- The CTS pin on any FTDI device will not be used, but must connect to a GND on the PCB.

```

Comment:
Technically the CTS doesn’t need to be connected but it can be.

Reply:
I was correcting it to read that the connection still needed to be made, as I didn't want people to think they had to pull the pin from the connector (and in fact that led to my near-overheat issue originally). Should I just remove that part entirely and just say to connect all wires as directed by the manufacturer since you match the expected pins already?

Reply:
Hmm maybe perhaps indicate they should pay extra attention to the ftdi cable and to not plug it in reverse, that the colours on the cable connectors match the labelled uart header pins. Perhaps a image of a ftdi (or even take a image of a cable plugged into the boards) and to make sure the red 5v is lined from the cable to the vcc pin on the uart headers? I will e-mail over some additional images.